aep/general/0231/aep.md.j2

```diff
@@ -1,5 +1,153 @@
 # Batch methods: Get
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/42) for more
-information.
+Some APIs need to allow users to get a specific set of resources at a
+consistent time point (e.g. using a read transaction). A batch get method
+provides this functionality.
+
+## Guidance
+
+APIs **may** support batch get to retrieve a consistent set of resources.
+
+- The method's name **must** begin with `BatchGet`. The remainder of the method
+  name **should** be the plural form of the resource being retrieved.
+- The HTTP verb **must** be `GET`.
+- The HTTP URI **must** end with `:batchGet`.
+- The URI path **should** represent the collection for the resource, matching
```

Comment:
Any reason this should can't be a must?

Reply:
Once again, this comes from the Google AIP. So what is the argument for making it "must"?